### PR TITLE
[FIX] Deployed filters

### PR DESCRIPTION
**Pull request by shivani170**
# Description
Added filters in `charts` >  `Deployed`  section
Screenshot 2021-06-09 at 6 56 11 PM

Fixes https://github.com/devtron-labs/dashboard/issues/136

### CODE REVIEW

File: src/components/charts/charts.util.tsx [hunk 1]
```diff
@@ -51,11 +51,29 @@ export const ValueContainer = props => {
         count = length
     }
 
+    function FilterName() {
```
Comment (rashmirai21) [inline src/components/charts/charts.util.tsx:54]: Don't use uppercase for functions which are not react components.

Comment (shivani170) [reply src/components/charts/charts.util.tsx]: fixed

**Review by pghildiyal — APPROVED** [review]

File: src/components/AppSelector/AppSelector.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const AppSelector: React.FC<{
     primaryKey: string; //url match
     primaryValue: string;
     matchedKeys: string[];
-    api: () => Promise<any>;
+    api: (qs) => Promise<any>;
```
Comment (rashmirai21) [inline src/components/AppSelector/AppSelector.tsx:12]: remove `qs`

Comment (shivani170) [reply src/components/AppSelector/AppSelector.tsx:12]: ok

Comment (shivani170) [reply src/components/AppSelector/AppSelector.tsx]: fixed

**Review by nishant-d — APPROVED** [review]

**Pull request merged**